src/features/apiDelegation/components/OverviewPageContent/OrgDelegationActionBar/OrgDelegationActionBar.tsx

```diff
@@ -95,7 +98,7 @@ export const OrgDelegationActionBar = ({
 
   return (
     <ActionBar
-      headingLevel={3}
+      headingLevel={4}
```

Comment:
```suggestion
      headingLevel={3}
```
Switching to h4 breaks the visible hierarchy of headings, as we already use h4 inside the action bars (and also means skipping the h3 level, which causes a warning in wave accessibility checks)